src/components/PredictionCircle.tsx

```diff
@@ -4,29 +4,38 @@ import React from "react";
 interface circleProps {
   largeCircle: boolean;
   predictionNumber: number;
+  isNormal: boolean;
 }
 
-const PredictionCircle = (props: circleProps) => {
-  const { largeCircle, predictionNumber } = props
-  let size = 120
-  let divSize = '100px'
+const PredictionCircle: React.FC<circleProps> = ({largeCircle, predictionNumber, isNormal}) => {
+  let divSize = '100px';
+
+  let predictionCircleAttributes = {
+    ariaDesc: "Prediction Result of Analysis",
+    constrainToVisibleArea: true,
+    data: { x: 'Prediction', y: predictionNumber },
+    height: 120,
+    width: 120,
+    padding: 0,
+    title: `${predictionNumber}%`,
+    thresholds: [{value: 0}, {value: 90}]
+  }
```

Comment:
I think it might be fine to just list these props directly in the ChartDonutUtilization component itself, since its only being used once.
I think it's also fine to just have a single `size` variable as opposed to writing out the `height` and `width` literals out, since we know the circles will have square dimensions.
Finally, one way we could simplify the threshold logic is to do
```
thresholds={isNormal ? [] : [{ value: 60 }, { value: 90 }]}
```
directly in the component props.

Let me know what you think of these suggestions.

Reply:
Agreeing on those suggestions.

Reply:
Will do, implementing those changes now